src/main/java/Duke.java

```diff
@@ -1,10 +1,140 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+    
+    private static void save(ArrayList<String> tasks) throws IOException {
+        File f = new File("tasks.txt");
+        System.out.print(f.createNewFile());
+        FileWriter fw = new FileWriter("tasks.txt");
+        for (String task : tasks) {
+            fw.write(task + System.lineSeparator());
+        }
+        fw.close();
+    }
+
+    private static void printFileContents(String filePath) throws FileNotFoundException {
+        File f = new File(filePath);
+        Scanner s = new Scanner(f);
+        while (s.hasNext()) {
+            System.out.println(s.nextLine());
+        }
+        s.close();
+    }
+
+    public static void main(String[] args) throws IOException {
+
+        Scanner scanner = new Scanner(System.in);
+
+        String logo = " _           _        \n"
+                + "| |    _   _| | _____ \n"
+                + "| |   | | | | |/ / _ \\\n"
+                + "| |___| |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+
+        ArrayList<String> list = new ArrayList<String>();
+        int amount = 0;
+        
+        String lineBreak = "\t____________________________________________________________";
+        System.out.println(lineBreak
+                + "\n\t Hello! I'm Luke, your slightly useful personal assistant!\n"
+                + "\t What can I do for you, my liege?\n"
+                + "\t Type 'list' to show previous inputs\n"
+                + "\t Type 'todo TASK' to indicate that TASK has to be done\n"
+                + "\t Type 'deadline TASK /by DATE/TIME' to indicate that TASK has to be done by DATE/TIME\n"
+                + "\t Type 'event TASK /at DATE/TIME PERIOD' to indicate that TASK occurs at DATE/TIME PERIOD\n"
+                + "\t Type 'done #' to indicate that task # has been done\n"
+                + "\t Type 'bye' to end\n"
+                + lineBreak);
+
+        String response = scanner.nextLine();
+
+        while (!response.equals("bye")) {

```

Comment:
I think you should try splitting the logic in this while loop into different files/methods